server/base.py

```diff
@@ -50,11 +50,40 @@ def _postUpload(event):
         Item.save(item)
 
 
+def checkForLargeImageFiles(event):
+    file = event.info
+    possible = False
+    mimeType = file.get('mimeType')
+    if mimeType in ('image/tiff', 'image/x-tiff', 'image/x-ptif'):
+        possible = True
+    exts = file.get('exts')
+    if isinstance(exts, list) and len(exts) >= 1 and exts[-1] in (
+            'svs', 'ptif', 'tif', 'tiff', 'ndpi'):
```

Comment:
Files are guaranteed to have the `exts` field which is always a list, so this could just be

    if exts and exts[-1] in (...):